11ty/_includes/layouts/base.njk

```diff
@@ -6,10 +6,24 @@
 
     {# Use title with path, or append a space to the page title to avoid collpasing with the meta title #}
     {% set pageTitle = titleWithPath or title + ' ' or '' %}
+    {% set alert = flag.level or '' %}
+    {% set preview = excerpt or renderData.description or description or metadata.description %}
+
     <title>{{ pageTitle + metadata.title }}</title>
+    <meta
+      name="twitter:description"
+      content="{{ alert + ": " + preview }}"
+    >
+    <meta name="twitter:card" content="summary"/>
+    <meta name="twitter:title" content="{{ pageTitle }}"/>
+    <meta
+      name="og:description"
+      content="{{ alert + ": " + preview }}"
+    >
+    <meta name="og:title" content="{{ pageTitle }}"/>
     <meta
       name="description"
-      content="{{ renderData.description or description or metadata.description }}"
+      content="{{ alert + ": " + preview }}"
```

Comment:
We can `{% set %}` this combination at the beginning of the file to avoid having to repeat this in `og:description`, `twitter:description` and `description`.